fixture-generator/Dockerfile

```
@@ -16,43 +16,49 @@ RUN apt-get update && apt-get install -y \
-    tar -xJ -C /usr/local --strip-components=1
+# Clone Kibana FIRST to get the .node-version file
+# Note: Kibana tags use "v" prefix (e.g., v9.2.0)
+ARG KIBANA_VERSION=v9.2.0
+WORKDIR /kibana
+RUN git clone --depth 1 --branch ${KIBANA_VERSION} https://github.com/elastic/kibana.git . || \
+    git clone --depth 1 --branch v${KIBANA_VERSION} https://github.com/elastic/kibana.git . || \
+    git clone --depth 1 https://github.com/elastic/kibana.git .
+
+# Install Node.js version specified by Kibana's .node-version file
+ARG TARGETARCH
+RUN NODE_VERSION=$(cat .node-version) && \
```

Hadolint (2.14.0)

[warning] Set the SHELL option -o pipefail before RUN with a pipe in it. If you are using /bin/sh in an alpine image or if your shell is symlinked to busybox then consider explicitly setting your SHELL to /bin/ash, or disable this check

(DL4006)